ui/app/components/tabs/tabs.component.js

```diff
@@ -0,0 +1,63 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+
+export default class Tabs extends Component {
+  static propTypes = {
+    defaultActiveTabIndex: PropTypes.number,
+    children: PropTypes.node,
+  }
+
+  constructor (props) {
+    super(props)
+
+    this.state = {
+      activeTabIndex: props.defaultActiveTabIndex || 0,
+    }
+  }
+
+  handleTabClick (tabIndex) {
+    const { activeTabIndex } = this.state
+
+    if (tabIndex !== activeTabIndex) {
+      this.setState({
+        activeTabIndex: tabIndex,
+      })
+    }
+  }
+
+  renderTabs () {
+    // const { children } = this.props
+    const numberOfTabs = React.Children.count(this.props.children)
```

Comment:
Out of curiosity why is `React.Children` (and not `this.props.children`) needed here?

Reply:
- [ ] remove commented out code

Reply:
`React.Children` is a set of utils that can be used on `this.props.children` https://reactjs.org/docs/react-api.html#reactchildren